packages/api/src/utils/zod.ts

```diff
@@ -279,12 +279,12 @@ export const translateExternalAlertToInternalAlert = (
     name: alertInput.name,
     message: alertInput.message,
     ...(alertInput.source === 'search' && alertInput.savedSearchId
-      ? { source: 'LOG', logViewId: alertInput.savedSearchId }
+      ? { source: 'LOG', savedSearchId: alertInput.savedSearchId }
```

Comment:
Ideally since we're migrating models we can just completely get rid of these functions or greatly simplify them right and just bake them into the model?

Reply:
Yeah I agree. So we don't need a translation layer. its not within the current scope and we will revisit it once we start working on public API